debian/build.py

```diff
@@ -46,11 +46,11 @@ def main():
   packages = {
     'kubectl': (
       ('1.3.7', '00'),
-      ('1.4.0-beta.3', '02'),
+      ('1.4.0-beta.5', '02'),
```

Comment:
Sure thing. Just didn't realise this was the case. May be we could convert this list to a dictionary, so it's more descriptive.